src/css/epicsevendb.scss

```diff
@@ -1463,14 +1463,10 @@ aside.column {
     display: flex;
     flex-direction: column;
 
-    .search-input {
-        width: 80%;
-        margin: 0 auto 1rem;
-        height: 1.2rem;
-        border-radius: 0.4rem;
-        padding: 1rem;
-        padding-top: 1.3rem;
+    .filter-by-name {
```

Comment:
I have not pulled to see how this looks, but I actually intentionally avoided the style used on the Hero/Artifact pages for search by name. I felt the textbox was too small to be in a place of prominence on the home page. What is your opinion?

Reply:
More specifically, I felt the smaller textbox didn't look good on a phone. That's also why I centered it instead of right-aligning it.

Reply:
![image](https://user-images.githubusercontent.com/1087109/53506899-aab46d00-3a95-11e9-8211-b379cb09ca26.png)

![image](https://user-images.githubusercontent.com/1087109/53506924-b99b1f80-3a95-11e9-8f17-22cbaa3e9cd5.png)


So, right now looks like this

Reply:
You don't think that looks too small on mobile?

Reply:
I kept same height you added.

your code actually had two (1.2rem and 3rem) 
![image](https://user-images.githubusercontent.com/1087109/53507532-d84de600-3a96-11e9-962f-7d7f7bc6bb25.png)

I just removed the 1.2 and kept the 3rem (which is 30px height).

Reply:
Oh, ok.  I thought you had removed everything.  Why no padding-top for vertical alignment?

Reply:
It's a css trick :P 
Setting `line-height` the same size as `height` already vertical-aligns the text inputted (and the placeholder), so no need vertical paddings. Only horizontal so the text doesn't touch borders.

Reply:
I did that originally, and it didn't look correct.. it was sitting too high.  Well, if it looks fine to you, that works for me.

Reply:
It looks higher because the entire phrase does not have descenders, that means, there's no `p q y g`, so the space on the font targeted for the descenders looks like white space.